### ISSUE TITLE
Move AWS test resource destruction to a shared library

### ISSUE THREAD

**Issue by seriva** [epiphany-platform/m-aws-kubernetes-service#31]
**Is your feature request related to a problem? Please describe.**
Currently both the `m-aws-kubernetes-service` and `m-aws-basic-infrastructure` modules have very similar code to clean up the resources afterwards.

**Describe the solution you'd like**
Make a go library which can be shared between modules to clean up the resources after running of the tests.

**Describe alternatives you've considered**
No

**Additional context**
No

**Comment by rafzei** [comment]
@seriva where this shared library should be placed?

**Comment by DaOsie** [comment]
In my opinion it should be placed in separate repo and used as a remote go module.

**Comment by seriva** [comment]
Yes that is exactly what I mean. Separate module which can be included remotely.
